src/main/webapp/app/shared/dashboards/tutor-participation-table/tutor-participation-table.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { TutorParticipationTableComponent } from 'app/shared/dashboards/tutor-participation-table/tutor-participation-table.component';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+
+@NgModule({
+    imports: [ArtemisSharedModule],
+    declarations: [TutorParticipationTableComponent],
+    exports: [TutorParticipationTableComponent],
```

Comment:
I'm not sure if defining a new module for this is really needed - from what I saw it is only used in the exercise dashboard, so adding it to the dashboard module might also make sense. This totally up for debate though, this is just my opinion